main.py

```diff
@@ -0,0 +1,162 @@
+from typing import Type
+
+from fastapi import (
+    FastAPI,
+    Depends,
+    HTTPException
+)
+from sqlalchemy.orm import Session
+
+import crud
+import models
+import schemas
+from database import SessionLocal, engine
+from models import Author, Book
+
+models.Base.metadata.create_all(bind=engine)
+
+app = FastAPI()
+
+
+def get_db():

```

Comment:
```suggestion
def get_db() -> None:
```
check annotation throughout the code twice before pushing